src/inspectors/FileInspector.ts

```diff
@@ -51,10 +49,13 @@ export class FileInspector implements IFileInspector {
   }
 
   private normalizePath(path: string): string {
+    if (!path) return path;
```

Comment:
It is maybe the convention, but IMHO it could be more apparent if it returns undefined than the undefined path.

Reply:
Removed. There was a problem with `@optional` inject. I have fixed it with non null assertion in initializer.